templates/insurance_companies/general_edit.html

```diff
@@ -90,6 +90,14 @@
             </div>
         </div>
     {/if}
+    {if $SUPPORT_ELIGIBILITY_REQUESTS}
+    <div class="form-group">
+        <label for="eligibility_id" class="control-label col-sm-2">{xl t='Payer Id For Eligibility'}</label>
```

Comment:
Smarty has a xlt function now :)
(note this won't work on the backport though and should keep xl for that)
```
<label for="eligibility_id" class="control-label col-sm-2">{xlt t='Payer Id For Eligibility'}</label>
```